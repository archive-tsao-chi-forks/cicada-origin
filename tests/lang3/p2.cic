z : { x: String y: String }
z = {
  x = "abc"
  y = "123"
  { x = x
    y = y }
}

@show z
@show z.x
@show z.y

w : String
w = {
  s = "www"
  s
}

@show w
