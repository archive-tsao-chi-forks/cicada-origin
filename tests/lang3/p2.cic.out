{
  x : String
  y : String
} -- {
  x = "abc"
  y = "123"
}
String -- "abc"
String -- "123"
String -- "www"
